Fliglio/Fli/DefaultResolverApp.php

```diff
@@ -55,13 +58,29 @@ public function configure(Configuration $cfg) {
 		foreach ($cfg->getInjectables() as $injectable) {
 			$this->addInjectable($injectable);
 		}
+	}
 
+	public function addAppFactory(AppFactory $appFactory) {
```

Comment:
Should this be `wrapDefaultInvokerApp` to support the possibility of injecting apps in other areas of the chain at a later date?  `addAppFactory` sounds too generic for how specific the implementation is